### PR TITLE
Remove set_default_dtype calls from jit and ops tests

### PR DESCRIPTION
**Pull request by kurtamohler**
Part of #68972

This only attempts to avoid setting the default dtype for `test_jit.py` and `test_ops.py`. There are other tests, like `test_nn.py`, which will be addressed in follow up PRs

cc @mruberry

### CODE REVIEW

**Comment by kurtamohler** [comment]
Note: I would like to add `assert torch.get_default_dtype() == torch.float32` to the `torch.testing._internal.common_utils.TestCase.setUp/tearDown` methods, but at the moment, that will make other tests (like `test_nn.py`) fail, since they still set the default to float64. I would prefer to break up this issue into several PRs to limit merge conflicts and CI failures and avoid getting stalled, so I will add the asserts in `setUp/tearDown` in a later PR

File: test/test_jit_cuda_fuser.py [hunk 1]
```diff
@@ -8,4 +11,6 @@ def run_tests():
     pass
 
 if __name__ == '__main__':
-    run_tests()
+    # TODO: Update nvfuser to work with float default dtype
```
Comment (ezyang) [inline test/test_jit_cuda_fuser.py:14]: TODO indeed lol

**Review by ezyang — APPROVED** [review]
I'm a bit skeptical about wrapping run_tests, since this won't apply if you don't go via that entry point (e.g., pytest)

**Pull request closed without merging**